tools/aa/AaTool.java

```diff
@@ -43,27 +47,13 @@ private void printUsage() {
   @Component(modules = {CommonModule.class, AaModule.class})
   public interface AaToolComponent {
     AaTool getAaTool();
-
-    @Component.Builder
-    interface Builder {
-      @BindsInstance
-      Builder configFileName(String configFileName);
-
-      AaToolComponent build();
-    }
-  }
-
-  @Inject
-  AaTool(InitCommand initCommand, WorkspaceCommand workspaceCommand) {
-    commands.put(initCommand.getName(), initCommand);
-    commands.put(workspaceCommand.getName(), workspaceCommand);
   }
 
   private void run(String[] args) {
     if (args.length > 0) {
       String command = args[0];
       if (commands.containsKey(command)) {
-        commands.get(command).run(args);
+        commands.get(command).get().run(args);
       } else {
         System.out.println("");
```

Comment:
Maybe needs to add some message instead **""**?

Reply:
It's a newline. The message is printed in `printUsage()`.